contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol

```diff
@@ -453,6 +459,14 @@ contract OracleReportSanityChecker is AccessControlEnumerable {
             revert IncorrectWithdrawalsVaultBalance(_actualWithdrawalVaultBalance);
     }
 
+    function _checkELRewardsVaultBalance(
+        uint256 _actualELRewardsVaultBalance,
+        uint256 _reportedELRewardsVaultBalance
+    ) internal pure {
+        if (_reportedELRewardsVaultBalance > _actualELRewardsVaultBalance)
```

Comment:
nitpicking alert. Please, add `{}` to multiline if-clauses.

Reply:
:+1: